server/src/main/java/keywhiz/KeywhizConfig.java

```diff
@@ -102,19 +101,12 @@
   private Long maximumSecretSizeInBytesInclusive;
 
   public enum RowHmacCheck {
-    DISABLED("disabled"),
-    DISABLED_BUT_LOG("logging"),
-    ENFORCED("enforced");
-
-    private final String configEntry;
-
-    RowHmacCheck(String configEntry) {
-      this.configEntry = configEntry;
-    }
-
-    public String getConfigEntry() {
-      return configEntry;
-    }
+    @JsonProperty("disabled")
+    DISABLED,
+    @JsonProperty("logging")
+    DISABLED_BUT_LOG,
+    @JsonProperty("enforced")
+    ENFORCED;
   }
```

Comment:
Short. And. Sweet.

Reply:
Does  the`@JsonProperty` annotation remove the need for getters and setters?

Reply:
For the enum, yes. We were using the configEntry field to model the serialized value of each enum value, but this can be accomplished more cleanly by using the Jackson annotation.